src/pages/Subscription/Subscription.jsx

```diff
@@ -0,0 +1,16 @@
+
+export default function SubscriptionPage(){
+	return (

```

Comment:
Looks like the indentation it's a little odd, let's install an extension on VS Code:

[EditorConfig for VS Code](https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig) - This is a way to configure the indentation and another configuration related to the Text Editor.

You need to create a file at the root of the project called `.editorconfig`,

Paste the most simple usage of it to define the correct indentation of the file(that will be 2 with spaces)

## .editorconfig
```
[*]
indent_style = space
indent_size = 4
```

Take a loot here to have more information about Editorconfig
https://editorconfig.org/